modules/nodes-nodes-rebooting-affinity.adoc

```diff
@@ -13,12 +13,8 @@ With this in place, if only two infrastructure nodes are available and one is re
 pod is prevented from running on the other node. `*oc get pods*` reports the pod as unready until a suitable node is available. 
 Once a node is available and all pods are back in ready state, the next node can be restarted.
```

Comment:
Can `podAntiAffinity` actually block further node reboots?  Linking [the upstream docs][1] too, in case that helps.  My impression is that it just limits pod scheduling, and if you also have a [pod disruption budget][2], perhaps the cluster would have to hold on further node reboots until sufficient evicted pods had been rescheduled and readied elsewhere.  Maybe this module is really about configuring robust workloads, and not specific to rebooting or nodes at all?

[1]: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity
[2]: https://kubernetes.io/docs/tasks/run-application/configure-pdb/